plugins/filesystem/local/local.php

```diff
@@ -37,11 +37,27 @@ class PlgFileSystemLocal extends CMSPlugin
 	 */
 	public function onFileSystemGetAdapters()
 	{
-		// Compile the root path
-		$filePath = ComponentHelper::getParams('com_media')->get('file_path', 'images');
-		$root = JPATH_ROOT . '/' . $filePath;
-		$root = rtrim($root) . '/';
+		$adapters = [];
+		$directories = $this->params->get('directories', []);
```

Comment:
The default value must be here also images, when not set to be consistent with the UI.

Reply:
The return value from subform is a json string and we parse it below. I'll fix it